src/main/web_src/src/pages/gruppe/PersonListe/modal/KommentarModal.js

```diff
@@ -0,0 +1,44 @@
+import DollyModal from '~/components/ui/modal/DollyModal'
+import React, { useState } from 'react'
+import { ErrorBoundary } from '~/components/ui/appError/ErrorBoundary'
+import ModalActionKnapper from '~/components/ui/modal/ModalActionKnapper'
+import { DollyApi } from '~/service/Api'
+import { Textarea } from 'nav-frontend-skjema'
+
+export const KommentarModal = ({ ident, closeModal }) => {
+	const lagreEndring = () => {
+		DollyApi.updateIdentBeskrivelse(ident.ident, kommentar).then(() => window.location.reload())
+	}
+
+	const [kommentar, setKommentar] = useState(ident.beskrivelse)
+	const MAX_LENGTH = 200
```

Comment:
Max lengde bør vel være samme som i det opprinnelige kommentarfeltet (1000)? 🤔

Reply:
Ah, vi var litt usikre på hvor lang den skulle være på zoom her om dagen, men da endrer jeg til det 😄 👍

Reply:
Jeg har ingen fasit på lengden altså, men tenker det viktigste er at det er samme max-lengde på det "vanlige" kommentarfeltet og det i modalen. For hvis man f.eks. hadde skrevet en kommentar på 400 tegn vil man jo få feil på feltet når man tar opp kommentar-modalen om max-lengden er 200 tegn der 😉  Og det kan jo hende folk allerede har skrevet ganske lange kommentarer, så jeg vil tro det er greit å beholde 1000 som max-lengde. Så slipper brukerne å forkorte kommentarene sine 😁

Reply:
Ja absolutt, ser at kommentarfeltet har 1000 på lengde ja, fant ikke det når jeg lette her om dagen 😅  Tenker jeg gjør litt endringer og importerer hele Beskrivelse komponenten inn i modal så blir det mindre repetering der🤓

Reply:
Ahaa! 😅  Det høres ut som en god ide 🤓